modules/helfi_paragraphs_news_list/helfi_paragraphs_news_list.module

```diff
@@ -56,7 +56,7 @@ function helfi_paragraphs_news_list_paragraph_view(
     $query = $storage
       ->getQuery();
     $query
-      ->condition('_language', $entity->language()->getId())
+      ->condition('search_api_language', $entity->language()->getId())
```

Comment:
Etusivu should reimplement _language so news_list paragraph does not break for instances that do not update immediately.

Reply:
[Added](https://github.com/City-of-Helsinki/drupal-helfi-etusivu/pull/743/commits/dd9b66cc34e6f7284a76aa8ed320a111b1836dcf)